Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -212,6 +214,18 @@ open class MediaControl: UICorePlugin, UIGestureRecognizerDelegate {
         }
     }
 
+    private func loadDefaultPlugins() {
```

Comment:
Heys guys,

According to Clappr architecture, we should not do this here. We had a [similar code during Loaders refactor](https://github.com/clappr/clappr-ios/pull/226#discussion_r223385017).

The loader is responsible for appending Plugins into the context not the context itself.